lib/fortitude/rails/renderer.rb

```diff
@@ -25,6 +25,11 @@ def render(widget_class, template_handler, local_assigns, is_partial, &block)
           end
 
           widget = widget_class.new(needed_assigns)
+
+          if options[:pathname]
+            widget.instance_variable_set(:@virtual_path, options[:pathname])
```

Comment:
In order for `ActionView::Helpers::CacheHelper` to work properly, the `@virtual_path` instance variable must be set: https://github.com/rails/rails/blob/215f86c3483cf300b2ccd6d5d8b97f9c9bf547f5/actionview/lib/action_view/helpers/cache_helper.rb#L230

See discussion in `template_handler.rb` below: